platform/web/os_web.cpp

```diff
@@ -160,6 +160,22 @@ bool OS_Web::_check_internal_feature_support(const String &p_feature) {
 	if (p_feature == "web") {
 		return true;
 	}
+
+	if (p_feature == "web_extensions") {
+#ifdef WEB_DLINK_ENABLED
+		return true;
+#else
+		return false;
+#endif
```

Comment:
Same here, `return false` is the default if there's no match in `has_feature` and related implementations.